rfd/0156-notifications.md

```diff
@@ -162,83 +186,87 @@ message ByRoles {
-   UserCreatedInformationalCode = "user-created-informational"
-   // UserCreatedWarningCode is the code for warning notifications that were manually created by a user using `tctl`.
-   UserCreatedWarningCode = "user-created-warning"
+   // UserCreatedInformationalType is the type for informational notifications that were manually created by a user using `tctl`.
+   UserCreatedInformationalType = "user-created-informational"
+   // UserCreatedWarningType is the type for warning notifications that were manually created by a user using `tctl`.
+   UserCreatedWarningType = "user-created-warning"
 
-   // AccessRequestPendingCode is the code for the notification of an access request pending review. This is the notification received
+   // AccessRequestPendingType is the type for the notification of an access request pending review. This is the notification received
    // by a user when an access request needs their review.
-   AccessRequestPendingCode = "access-request-pending"
-   // AccessRequestApprovedCode is the code for the notification of a user's access request being approved.
-   AccessRequestApprovedCode = "access-request-approved"
-   // AccessRequestDeniedCode is the code for the notification of a user's access request being denied.
-   AccessRequestDeniedCode = "access-request-denied"
+   AccessRequestPendingType = "access-request-pending"
+   // AccessRequestApprovedType is the type for the notification of a user's access request being approved.
+   AccessRequestApprovedType = "access-request-approved"
+   // AccessRequestDeniedType is the type for the notification of a user's access request being denied.
+   AccessRequestDeniedType = "access-request-denied"
 )
 ```
 
-For example, if a notification's code is `access-request-pending`, the UI will make the notification redirect to the page for reviewing the access request.
+For example, if a notification's type is `access-request-pending`, the UI will make the notification redirect to the page for reviewing the access request.
 
 #### Usage
 
-The notifications service will work primarily by adding, deleting, and fetching notifications from the database. There will also be a watcher which
+The notifications service will work primarily by adding, deleting, and fetching notifications from the backend. There will also be a watcher which
 watches for any changes to the database in order to perform real-time actions when a new notification is added.
 
 These are some examples of methods of the service which can be called:
 
 ```go
-// Creates a user-specific notification for a user. This will be stored in the database as a Notification under `/notifications/user/<cluster id>/<auth connector>/<username>/<notification.id>`.
+// Creates a user-specific notification for a user. This will be stored in the backend as a Notification under `/notifications/user/<auth connector>/<username>/<notification.id>`.
```

Comment:
Similar comment here. I think we're over-specifying at this point.

Reviewers can get a good enough idea if you just describe the types of operations the service will expose without putting in the actual function signatures (which are likely to change).